docs: refer to the web app as "W&B" instead of "W&B App" (#2737)
wandb/docs: models/artifacts/track-external-files.mdx

For AWS, if the bucket is not located in the configured user's default region, you must set the `AWS_REGION` environment variable to match the bucket region.

<Warning>

> Rich media such as images, audio, video, and point clouds may fail to render in the App UI depending on the CORS configuration of your bucket. Allow listing **app.wandb.ai** in your bucket's CORS settings will allow W&B to properly render such rich media.

Reviewer: ```suggestion
Rich media such as images, audio, video, and point clouds may fail to render in the App UI depending on the CORS configuration of your bucket. Allow-list `app.wandb.ai` in your bucket's CORS settings so W&B can properly render such rich media.
```
The next paragraph may be redundant.
